src/app/api/auth/config.ts

```diff
@@ -19,13 +19,18 @@ export const authOptions: NextAuthOptions = {
       authorization: { params: { scope: "openid profile email elixir_id" } },
     }),
   ],
+  session: {
```

Comment:
important: we must work on this in another PR, as we must understand the implications  behind these changes.

https://github.com/GenomicDataInfrastructure/gdi-userportal-frontend/issues/355